src/popup/pages/FungibleTokens/TokenContainer.vue

```diff
@@ -212,19 +229,57 @@ export default defineComponent({
 
     const convertedBalance = computed(() => +tokenData.value.convertedBalance! || 0);
 
+    function calculateRouterHeight() {
+      nextTick(() => {
+        const ionicWrapperBottom = document.querySelector('.app-wrapper')?.getBoundingClientRect()?.bottom;
+        const tabsWrapperElementBottom = stickyTabsWrapperEl.value?.getBoundingClientRect().bottom;
+        routerHeight.value = `${ionicWrapperBottom! - tabsWrapperElementBottom!}px`!;
+      });
+    }
+
+    /**
+     * Observe tabs wrapper height changes and recalculate router height.
+     * Tabs change height when filters are shown/hidden
+     */
+    function observeTabsWrapperHeight() {
+      const resizeObserver = new ResizeObserver(() => {
+        calculateRouterHeight();
+      });
+      resizeObserver.observe(stickyTabsWrapperEl.value!);
+    }
+
     onMounted(async () => {
       if (isContract(contractId) && !isAe) {
         await getAeSdk();
         tokenPairs.value = await store.dispatch('fungibleTokens/getContractTokenPairs', contractId);
       }
       loading.value = false;
+      setTimeout(() => {
+        observeTabsWrapperHeight();
+        calculateRouterHeight();
+      }, 250);
+    });
+
+    onIonViewDidEnter(() => {
+      setTokenProps({
+        contractId: unref(contractId),
+        tokenPairs: unref(tokenPairs),
+        tokenData: unref(tokenData),
+        tokens: unref(tokens),
+        isMultisig: unref(isMultisig),
```

Comment:
Using `unref` is fully correct, but I think it's more common (at least in our repo) to just add `.value` in order to get real value.